<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplesWidget</class>
 <widget class="QWidget" name="SamplesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,1,0,0,0,0,1" columnstretch="1,0,0,0,1,0,0,0,1">
   <item row="3" column="5">
    <widget class="SampleWidget" name="sampleWidget_13" native="true"/>
   </item>
   <item row="6" column="2">
    <widget class="SampleWidget" name="sampleWidget_11" native="true"/>
   </item>
   <item row="2" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="SampleWidget" name="sampleWidget_4" native="true"/>
   </item>
   <item row="3" column="2">
    <widget class="SampleWidget" name="sampleWidget_2" native="true"/>
   </item>
   <item row="4" column="7">
    <widget class="SampleWidget" name="sampleWidget_18" native="true"/>
   </item>
   <item row="3" column="8">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="5">
    <widget class="SampleWidget" name="sampleWidget_22" native="true"/>
   </item>
   <item row="5" column="5">
    <widget class="SampleWidget" name="sampleWidget_19" native="true"/>
   </item>
   <item row="5" column="2">
    <widget class="SampleWidget" name="sampleWidget_8" native="true"/>
   </item>
   <item row="3" column="3">
    <widget class="SampleWidget" name="sampleWidget_3" native="true"/>
   </item>
   <item row="5" column="7">
    <widget class="SampleWidget" name="sampleWidget_21" native="true"/>
   </item>
   <item row="6" column="7">
    <widget class="SampleWidget" name="sampleWidget_24" native="true"/>
   </item>
   <item row="5" column="1">
    <widget class="SampleWidget" name="sampleWidget_7" native="true"/>
   </item>
   <item row="6" column="3">
    <widget class="SampleWidget" name="sampleWidget_12" native="true"/>
   </item>
   <item row="4" column="2">
    <widget class="SampleWidget" name="sampleWidget_5" native="true"/>
   </item>
   <item row="4" column="6">
    <widget class="SampleWidget" name="sampleWidget_17" native="true"/>
   </item>
   <item row="3" column="7">
    <widget class="SampleWidget" name="sampleWidget_15" native="true"/>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="6">
    <widget class="SampleWidget" name="sampleWidget_20" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="SampleWidget" name="sampleWidget_1" native="true"/>
   </item>
   <item row="4" column="3">
    <widget class="SampleWidget" name="sampleWidget_6" native="true"/>
   </item>
   <item row="4" column="5">
    <widget class="SampleWidget" name="sampleWidget_16" native="true"/>
   </item>
   <item row="6" column="6">
    <widget class="SampleWidget" name="sampleWidget_23" native="true"/>
   </item>
   <item row="3" column="6">
    <widget class="SampleWidget" name="sampleWidget_14" native="true"/>
   </item>
   <item row="7" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="SampleWidget" name="sampleWidget_10" native="true"/>
   </item>
   <item row="5" column="3">
    <widget class="SampleWidget" name="sampleWidget_9" native="true"/>
   </item>
   <item row="1" column="0" colspan="9">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Sequencer</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="SequencerWidget" name="sequencerWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Swap left/right</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStopAll">
       <property name="text">
        <string>Stop all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SampleWidget</class>
   <extends>QWidget</extends>
   <header>samplewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SequencerWidget</class>
   <extends>QWidget</extends>
   <header>sequencerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
